mailpoet/assets/js/src/segments/dynamic/templates/components/template_list_item.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  Tooltip,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { starFilled } from '@wordpress/icons';
+import { Tag } from '@woocommerce/components';
+
+type TemplateListItemProps = {
+  template: {
+    id: number;
+    name: string;
+    description: string;
+    isEssential: boolean;
+    isEngaged: boolean;
+  };
+};
+
+export function TemplateListItem({
+  template,
+}: TemplateListItemProps): JSX.Element {
+  return (
+    <Card className="mailpoet-templates-card">
+      <CardHeader className="mailpoet-templates-card-header">
+        {template.isEssential && (
+          <Tooltip text={__('Essential segment', 'mailpoet')}>
```

Comment:
My preference would be to include a `delay` prop of 0 to make these appear immediately. I'm not sure what design had in mind.

Reply:
I don't have a strong preference here, but I'd incline to use the default value of 700 milliseconds for `delay` instead of changing it to `0`. Or is there an indication in the Figma file or somewhere else that I missed that we should change this to `0`?

Reply:
Strictly a personal preference, so feel free to leave it as is, especially since this would be the default behavior for tooltips in general.

I didn't think it was working at first because I wasn't seeing it, but I just wasn't being patient enough. I don't think it's really obvious that the star is a tooltip trigger, so I was thinking people might miss it.

I think there may be some accessibility issues with the tooltip itself, but that's just the tooltip component itself and not something we can/should address in this PR. The only thing I think we could maybe change is I believe tooltip triggers should be focusable with the keyboard, but that wouldn't be the greatest UX since you would have to tab through the same tooltip multiple times in a listing with multiple essential segments.

There's other stuff too, like how I don't think the container for the tooltip content should have `aria-hidden=true`, there's no `role="tooltip` or `aria-describedby`, and it can't be dismissed with the keyboard other than by focusing something else (I read that you should be able to dismiss them with something like the esc key), but again these are all (potential) issues with the core Tooltip component itself, and I could be wrong! I'm no accessibility expert.

All that being said, this isn't exactly vital information for a user, so I don't think it's very important that these tooltips be perfectly accessible.

Reply:
Thanks for sharing all your thoughts about the potential accessibility issues for this component. I agree that this is not something we should address in this PR.